database/mysql/mysql_user.py

```diff
@@ -341,7 +341,7 @@ def privileges_unpack(priv):
 
     # if we are only specifying something like REQUIRESSL in *.* we still need
     # to add USAGE as a privilege to avoid syntax errors
-    if priv.find('REQUIRESSL') != -1 and 'USAGE' not in output['*.*']:
+    if priv.find('REQUIRESSL') != -1 and not filter(lambda x: x not in [ 'REQUIRESSL' ], output['*.*']):
```

Comment:
This code isn't at all clear to me. 

I think it's saying 
```
if 'REQUIRESSL' in priv and 'REQUIRESSL' not in output['*.*']:
```
but I'm not sure how that would happen, and don't we still want to check `'USAGE' not in output['*.*']`